sunpy/data/__init__.py

```diff
@@ -1,3 +1,41 @@
+import astropy.units as u
+import sys
```

Comment:
import ordering, should be stdlib, line break, astropy. (you can also run isort).